drv/gimlet-hf-server/src/main.rs

```diff
@@ -135,6 +135,8 @@ fn main() -> ! {
     let mut server = ServerImpl {
         qspi,
         block: [0; 256],
+        // Capacity is stored as log2 size
+        capacity: 1 << capacity,
```

Comment:
Not your changes, but it would be nice if the first `capacity` was named `log2_capacity` here, for clarity

Reply:
Agreed, plus that would eliminate the need for the comment :+1: